compiler/test/stdlib/option.test.gr

```diff
@@ -163,17 +163,19 @@ let mut d = "🌾"
 assert Option.peek(x => fail "Shouldn't be called", None) == None
 assert d == "🌾"
 
-// Option.or
-assert Option.or(Some(1), Some(2)) == Some(1)
-assert Option.or(Some(1), None) == Some(1)
-assert Option.or(None, Some(2)) == Some(2)
-assert Option.or(None, None) == None
-
-// Option.and
-assert Option.and(Some(1), Some(2)) == Some(2)
-assert Option.and(Some(1), None) == None
-assert Option.and(None, Some(2)) == None
-assert Option.and(None, None) == None
+from Option use { (||), (&&) }
+
+// Option.(||)
+assert (Some(1) || Some(2)) == Some(1)
```

Comment:
Is there a reason you left the parens here, I think this will fail the formatter test.
Same for all the other tests you replaced